server/block/frosted_ice.go

```diff
@@ -0,0 +1,131 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/item/enchantment"
+	"github.com/go-gl/mathgl/mgl64"
+	"math/rand"
+	"time"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+// FrostedIce is a translucent solid block. It is generated by players walking
+// near water while wearing boots enchanted with frost walker.
+type FrostedIce struct {
+	solid
+	
+	// Age is the melting state of frosted ice. Values range from 0 to 4.
+	// When the age is 4 however, the block melts completely into water.
+	Age int
+}
+
+// BreakInfo ...
+func (fi FrostedIce) BreakInfo() BreakInfo {
+	effective := func(_ item.Tool) bool { return true }
+
+	drops := func(_ item.Tool, _ []item.Enchantment) []item.Stack { return []item.Stack{} }
+
+	bi := newBreakInfo(0.5, alwaysHarvestable, effective, drops)
```

Comment:
```suggestion
	bi := newBreakInfo(0.5, alwaysHarvestable, alwaysEffective, simpleDrops())
```

Reply:
alwaysEffective does not exist, I can create it though?

Reply:
nothingEffective*